kolibri/core/errorreports/models.py

```diff
@@ -48,8 +48,6 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
 
 
 class ErrorReports(models.Model):
-    FRONTEND = "frontend"
-    BACKEND = "backend"
     POSSIBLE_ERRORS = [
```

Comment:
can we move this to the constants file too?

Reply:
Done